woocommerce/woocommerce
# [CYS] Update footer and homepage copy

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR updates the footer and homepage sections of the assembler sidebar to match the designs.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Ensure the `WooCommerce Beta Tester` plugin is installed and activated (available on this monorepo).
2. Head over to `/wp-admin/tools.php?page=woocommerce-admin-test-helper` and enable `customize-store` feature flag.
3. Go to `WooCommerce > Home`, click on `Start customizing`, and finish the flow.
4. Check the copy matches the screenshots below for Core and WooExpress:

| Core | WooExpress |
|--------|--------|
| <img width="347" alt="Screenshot 2024-02-22 at 10 00 30" src="https://github.com/woocommerce/woocommerce/assets/186112/08764aba-a712-402b-aab2-c35e6f35066d"> | <img width="402" alt="design-x" src="https://github.com/woocommerce/woocommerce/assets/186112/a4902dd2-1afc-4d73-b3b5-5c3e2a9623c6"> |
| <img width="404" alt="footer-core" src="https://github.com/woocommerce/woocommerce/assets/186112/88601983-0091-4bfd-b3f2-ed5c1159b640"> |<img width="357" alt="footer-x" src="https://github.com/woocommerce/woocommerce/assets/186112/74ef98c5-8f8a-4058-9992-d9b153e61a08">  | 



### Changelog entry





<details>

#### Significance




#### Type




#### Message 
CYS - Updates the footer and homepage sections of the assembler sidebar.
#### Comment 

</details>
